views/tx.tmpl

```diff
@@ -18,7 +18,11 @@
                     <th>Confirmations</th>
                     <th>Included in block</th>
                     <th>Type</th>
-                    <th>Size</th>
+                    {{if .FeeRate}}
+                    <th>Fee</th>
+                    <th>Fee Rate</th>
+                    {{end}}
+                    <th>Size {{(index .Vin 0).Coinbase}}</th>
```

Comment:
I don't follow this.  Can you explain what this should show?

Reply:
That should not be there